vite/src/keycloak.ts

```diff
@@ -0,0 +1,31 @@
+import Keycloak from "keycloak-js";
+
+const initOptions = {
+  url: import.meta.env.VITE_KEYCLOAK_URL,
+  realm: import.meta.env.VITE_KEYCLOAK_REALM,
+  clientId: import.meta.env.VITE_KEYCLOAK_CLIENT_ID
+}
+
+const keycloakInstance = new Keycloak(initOptions);
+
+interface CallbackOneParam<T1 = void, T2 = void> {
+  (param1: T1): T2;
+}
+
+const Login = (onAuthenticatedCallback: CallbackOneParam) => {
+  keycloakInstance
+    .init({ onLoad: "login-required" })
+    .then(function (authenticated) {
+      authenticated ? onAuthenticatedCallback() : alert("non authenticated");
+    })
+    .catch((e) => {
+      console.dir(e);
+      console.log(`keycloak init exception: ${e}`);
+    });
+};
+
+const KeyCloakService = {
+  CallLogin: Login,
+};
+
+export default KeyCloakService;
```

Comment:
```suggestion
export default KeyCloakService;

```
ファイルの最後の行は改行でお願いします。